src/Dialect/ONNX/Transforms/Decompose.td

```diff
@@ -354,6 +354,11 @@ def ClipV12Pattern : Pat<
   (ONNXClipOp $x, $min, $max)
 >;
 
+def DFTV17Pattern : Pat<
+  (ONNXDFTV17Op $x, $dft_length, $axis, $inverse, $onesided),
+  (ONNXDFTOp $x, $dft_length, (ONNXConstantOpFromDenseAttr(createScalarDenseAttrRank0 $axis)), $inverse, $onesided)
+>;
+
```

Comment:
Could you add a lit test for this decomposition?

Reply:
Okay I can add a test